src/importexport/musicxml/internal/musicxml/importmxmlpass1.h

```diff
@@ -175,6 +177,12 @@ class MusicXMLParserPass1
     int octaveShift(const QString& id, const staff_idx_t staff, const Fraction f) const;
     const CreditWordsList& credits() const { return _credits; }
     bool hasBeamingInfo() const { return _hasBeamingInfo; }
+    bool isVocalStaff(const QString& id) const { return _parts[id].isVocalStaff(); }
+    static VBox* createAndAddVBoxForCreditWords(Score* const score, const int miny = 0, const int maxy = 75);
+    const int maxDiff() { return _maxDiff; }
```

Comment:
not rather `int maxDiff() const { return _maxDiff; }`? Otherwise there's a warning "'const' type qualifier on return type has no effect"